recipes/r-bit/meta.yaml

```diff
@@ -80,8 +80,8 @@ about:
 # Title: A class for vectors of 1-bit booleans
 # Version: 1.1-12
 # Date: 2012-01-17
-# Author: Jens Oehlschlägel <Jens.Oehlschlaegel@truecluster.com>
-# Maintainer: Jens Oehlschlägel <Jens.Oehlschlaegel@truecluster.com>
+# Author: Jens Oehlschlaegel <Jens.Oehlschlaegel@truecluster.com>
```

Comment:
@johanneskoester never mind. I just realized when trying to reproduce locally that these errors were due to me having an older local conda-build version built from a git revision.  I will revert these ascii-related recipes back to master.